src/prof/src/http/templates/flamegraph.html

```diff
@@ -1,4 +1,4 @@
-{% extends "base.html" %}
+{% extends "../../materialized/src/http/templates/base.html" %}
```

Comment:
Reaching into the `materialized` crate here is a bit backwards, since `materialized` depends on this crate. I'd advocate for creating a _new_ base.html file that thinks of itself as a separate "pprof" project/product. This is how the official pprof project works: https://github.com/google/pprof/blob/master/internal/driver/html/top.html. You just get what you get with the UI. Keeps concerns nice and separate, and we don't care about having an integrated UI experience here since this is just a dev tool.